webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentVersion.js

```diff
@@ -10,33 +10,46 @@ import {
 } from '@patternfly/react-core';
 
 const ComponentVersion = ({ componentCV }) => {
-  const { latest, content_view_version: cvVersion, content_view: cv } = componentCV;
+  const {
+    id: componentId, latest, content_view_version: cvVersion, content_view: cv,
+  } = componentCV;
   const {
     id,
     latest_version: latestVersion,
   } = cv;
-  const { version } = cvVersion;
+  const { version } = cvVersion || {};
+  const noVersionText = __('Not yet published');
   const latestDescription = __('Latest (automatically updates)');
   const manualVersionText = (latestVersion === version) ? __('Latest version') : __(`New version is available: Version ${latestVersion}`);
+  if (componentId) {
+    return (
+      <>
+        <Link to={urlBuilder('labs/content_views', '', id)}>{version ? `Version ${version}` : noVersionText}</Link>
+        <TextContent>
+          <Text component={TextVariants.small}>
+            {latest ? latestDescription : manualVersionText}
+          </Text>
+        </TextContent>
+      </>
+    );
+  }
   return (
     <>
-      <Link to={urlBuilder('labs/content_views', '', id)}> Version {version}</Link>
-      <TextContent>
-        <Text component={TextVariants.small}>{latest ? latestDescription : manualVersionText}</Text>
-      </TextContent>
+      <Link to={urlBuilder('labs/content_views', '', id)}>{latestVersion ? `Version ${latestVersion}` : noVersionText}</Link>
     </>
```

Comment:
Remove unneeded fragments

Reply:
Updated to remove the unnecessary fragments. 👍🏽